### ISSUE TITLE
Location of a shared task should not be displayed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Kofil/moloko#12]
```
When I recieve a task that is shared from one of my contacts, if the task had a 
location for this contact, RTM site doesn't show the location of the task but 
Moloko shows a location of Lat 0.0, Lon 0.0

Way to reproduce:
1. have a contact with a task that have a location
2. have this task shared with me
3. look at this task on RTM site : no location is set for it
4. sync Moloko and look at the shared task
5. Moloko shows a task with Lon 0.0, Lat 0.0 location whereas it should not 
show any location.
```

Original issue reported on code.google.com by `david.ve...@gmail.com` on 7 Dec 2010 at 11:44

**Comment by GoogleCodeExporter** [comment]
```
RTM gives shared tasks with the location ID of the "task owner" context. This 
location ID is not available in the context of the receiver of a shared task 
cause the location itself is not shared.

This has to be handled in Moloko.
```

Original comment by `ronny.roehricht` on 7 Dec 2010 at 2:14
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in r175 / 0.9.6b
```

Original comment by `ronny.roehricht` on 7 Dec 2010 at 7:02
- Changed state: **Fixed**
